tests/unit/airlock/test_views.py

```diff
@@ -136,12 +141,57 @@ def test_api_post_release_request_custom_org_and_repo(mock_create_issue, api_rf)
     ]
 
 
+@patch("airlock.views._get_github_api", FakeGitHubAPI)
+@patch("airlock.views.create_output_checking_issue")
+def test_api_post_release_request_non_default_org_and_repo(mock_create_issue, api_rf):
```

Comment:
It might be worth using the `settings` fixture and defining a test-specific org in `ORG_OUTPUT_CHECKING_REPOS`. Otherwise this test is dependent on the actual settings for bristol and it'll break if they ever change their output checking repo

Reply:
Good point.